traffic_ops/traffic_ops_golang/federation_resolvers/federation_resolvers.go

```diff
@@ -301,3 +306,153 @@ func deleteFederationResolver(inf *api.APIInfo) (tc.Alert, tc.FederationResolver
 
 	return alert, result, userErr, sysErr, statusCode
 }
+
+// GetFederationResolvers [V5] - get list of federation resolver for APIv5
```

Comment:
Since federation_resolvers is not using the CRUDer interface, it will be much easier (and would involve lesser lines of code) if we could just check the api version in the `Get` and `Create` methods of federation_resolvers, and return the appropriate structs. 
You can take a look at this PR for reference: https://github.com/apache/trafficcontrol/pull/7547

Reply:
nice, i would do this srijeet